components/MyMessage.vue

```diff
@@ -189,14 +189,24 @@
                   message.location &&
                   message.item
                 "
-                variant="secondary"
-                disabled
-                class="mr-2 mb-1"
-                title="You will be able to repost this soon"
+                class="position-relative"
               >
-                <v-icon class="d-none d-sm-inline" icon="sync" /> Repost
-                <span class="small">{{ timeago(message.repostat) }}</span>
-              </b-button>
+                <b-button
+                  variant="secondary"
+                  class="mr-2 mb-1"
+                  title="You will be able to repost this soon"
+                  @click.stop="repostWhenUnavailable"
+                >
+                  <v-icon class="d-none d-sm-inline" icon="sync" /> Repost
+                  <span class="small">{{ timeago(message.repostat) }}</span>
+                </b-button>
+                <p
+                  class="invalid-feedback position-absolute bg-white text-center"
+                  :class="triedToRepost ? 'd-block' : 'd-none'"
```

Comment:
Does it work to have two values of the `class` attribute?  I'm not sure.

Reply:
You mean both `class` and `:class`? For sure yes.

Reply:
https://vuejs.org/guide/essentials/class-and-style.html#binding-html-classes

Third example by the link.

Reply:
> You mean both `class` and `:class`? For sure yes.

Good to know!  There are quite a few places where I wish I'd done that.